foo_amr/foo_input_amr.cpp

```diff
@@ -309,7 +309,7 @@ const unsigned input_amr::m_start = 7;
  * @{
  */
 static input_singletrack_factory_t<input_amr> g_input_amr_factory;
-DECLARE_COMPONENT_VERSION("AMR input","1.1.1","http://angelo.scene.pl/\n(c) 2003-2018, Andrzej Lichnerowicz\nPowered GSM AMR-NB speech codec\n(c) 2001, 3gpp");
+DECLARE_COMPONENT_VERSION("AMR input","1.1.2","https://github.com/unjello/foo_input_amr/; 2003-2018: Andrzej Lichnerowicz, Quang Nguyen\nPowered GSM AMR-NB speech codec\n2001, 3gpp");
```

Comment:
Hey, sorry to pick again, but the AMR-NB codec is still copyrighted, to the latter `(c)` should stay.

Reply:
Yep :+1: . After changing files, I realized that AMR-NB codec is still copyrighted. I have recovered (c). 
Please review my commit here: 
https://github.com/unjello/foo_input_amr/pull/1/commits/c92f7ade4526bd613aaf91c98b41031baf0daf33